### DISCUSSION TITLE
Cannot connect to SB 4000TL-21 Inverters using node-red-contrib-sma-webconnect

### DISCUSSION THREAD

**Discussion by fritserasmus** [rlindner/node-red-contrib-sma-webconnect#32] [Q&A]
I deployed the node in my Node-RED install and the Sunny Boy 4000TL-21 is not available as an option and neither can Node_Red establish a connection to the SB option available.

Is there anything that I have not done correctly, or does this Node only connect to the Tripower and  SB 2.5?

Kind Regards

Frits

**Comment by rlindner** [comment]
The node comes with predefined configurations for the mentioned models, but you can provide an own configuration for other models like described in the readme: https://github.com/rlindner/node-red-contrib-sma-webconnect#read-custom-values
